tests/unit/dogstatsd/test_statsd.py

```diff
@@ -421,6 +421,27 @@ def test_tags_from_environment_and_constant(self):
         statsd.gauge('gt', 123.4)
         t.assert_equal('gt:123.4|g|#country:canada,red,country:china,age:45,blue', statsd.socket.recv())
 
+    def test_gauge_doesnt_send_None(self):
+        self.statsd.gauge('metric', None)
+        assert self.recv() is None
+
+    def test_increment_doesnt_send_None(self):
+        self.statsd.increment('metric', None)
+        assert self.recv() is None
+
+    def test_decrement_doesnt_send_None(self):
+        self.statsd.decrement('metric', None)
+        assert self.recv() is None
+
+    def test_timing_doesnt_send_None(self):
+        self.statsd.timing('metric', None)
+        assert self.recv() is None
+
+    def test_histogram_doesnt_send_None(self):
+        self.statsd.histogram('metric', None)
+        assert self.recv() is None
+
```

Comment:
I typically do one assertion per test -- then if the test fails, you know from the test name alone what the problem is. If you guys use a different style, I can definitely consolidate them, let me know what your preference is.